pipelines/algolia_hn_search/__init__.py

```diff
@@ -0,0 +1,127 @@
+"""
+	Loads all the search hits for specific keywords from the Algolia Hacker News Search API
+	https://hn.algolia.com/api
+"""
+import dlt
+import time
+
+from datetime import datetime
+from dlt.sources.helpers import requests
+
+
+url = "https://hn.algolia.com/api/v1/search_by_date"
+
+@dlt.source
+def algolia_hn_search(
+	keywords, # list of keywords for which the data needs to be loaded
+	start_date, # start date in datetime or "yyyy-mm-dd" format
```

Comment:
there's a conceptual problem: where the `start_date` is coming from on subsequent loads? user is supposed to remember it themselves? if you want to convert your source to incremental, I'll show you how to do it below